src/utils/update/trust.model.fetch.ts

```diff
@@ -319,14 +321,8 @@ const mapTrusteeType = (trusteeTypeId: string): RoleWithinTrustType | undefined
       case "5002":
         return RoleWithinTrustType.BENEFICIARY;
       case "5001":
-        /*
-          Type 5001 is not a valid type for an individual or corporate trustee. Instances of individual or corporate
-          trustees that have this trusteeTypeId are the result of incorrect data originally submitted through the
-          trust spreadsheet (which has now been superseded by web screens). In these instances we do not display the type.
-          Data fixes should prevent these instances from being retrieved.
-        */
-        logger.info(`Warning - invalid data. Trustee type ${trusteeTypeId} found when mapping trustee data`);
-        return undefined;
+        // Type 5001 is only a valid trustee type for a historical beneficial owner.
```

Comment:
Maybe don't need this comment? The other mappings don't have one.

Reply:
Removed comment